### ISSUE TITLE
Implement activation functions

### ISSUE THREAD

**Issue by divij-zenteiq-ai** [zenoxml/SciREX#107]
# Backend agnostic implementation for the deep learning submodule

Common activation functions (`tanh`, `sigmoid`, `relu`, etc) need to be implemented across the `tensorflow` and `jax` backends to be used in the deep learning codes. 

These functions will share names across backends, to be used by the deep learning module agnostically.

For example, for the `tensorflow` backend, we can write
```python
def tanh(x):
    return tf.nn.tanh(x)
```
